proto/client/gClient.go

```diff
@@ -172,7 +172,6 @@ func (confClient *ConfigClient) subscribeToConfigPod(commChan chan *protos.Netwo
 				}
 			} else if status == connectivity.Idle {
 				logger.GrpcLog.Errorf("Connectivity status idle, trying to connect again")
-				confClient.Conn.Connect()
```

Comment:
Why deleting this and not replacing it for an appropriate method/function to handle the case when `status == connectivity.Idle`?
I understand that this gets deleted otherwise `go build` will fail with the message shown below. However, if we delete this line, what is the purpose of having the `else if` statement?
```shell
# github.com/omec-project/config5g/proto/client
proto/client/gClient.go:175:21: confClient.Conn.Connect undefined (type *grpc.ClientConn has no field or method Connect)
```

Reply:
I mean, connect didn't do anything in the first place, here we simply remove it. The goal of theses PR's is not to change their behavior but to make sure they follow lint standards. I agree that we can re-think the code we have here as elsewhere but I would scope this out of linting improvements.

Reply:
Is there a way to leave a note about this such that we do not forget about clean up this stuff. For example, commenting out instead of deleting it, or some other way. The idea would be to keep track this thing to clean it up in the future?
```diff
-				confClient.Conn.Connect()
+				// confClient.Conn.Connect()
```

Reply:
I don't mind adding a note in the code but I'm doubtful of this as a strategy to keep track of what has to be done.